libgraph/include/katana/TransformationView.h

```diff
@@ -0,0 +1,105 @@
+#ifndef KATANA_LIBGRAPH_KATANA_TRANSFORMATIONVIEW_H_
+#define KATANA_LIBGRAPH_KATANA_TRANSFORMATIONVIEW_H_
+
+#include <memory>
+#include <utility>
+
+#include "katana/PropertyGraph.h"
+
+namespace katana {
+
+/// A TransformationView is a topological transformation of the property graph.
+/// It is derived from the PropertyGraph so that it can be treated by query and
+/// analytics routines as a regular property graph.
+class KATANA_EXPORT TransformationView final : public PropertyGraph {
+public:
+  TransformationView() = default;
+  TransformationView(TransformationView&& other) = default;
+
+  virtual ~TransformationView();
+
+  /// Make a projected graph from a property graph. Shares state with
+  /// the original graph.
+  static std::unique_ptr<TransformationView> MakeProjectedGraph(
```

Comment:
rename this to `MakeTransformationView`

Reply:
I disagree. This method creates a TransformationView, but it's not the only kind that could be created. A transformation view isn't exclusively for projections.  There could be others, so this should be named to match the kind of graph we are creating.

Reply:
sure